app/controllers/api/articles_controller.rb

```diff
@@ -49,4 +49,20 @@ def show
       message: 'Article not found.'
     }, status: 404
   end
+
+  def create
+    current_admin.articles.create(
+      title: params[:title], teaser: params[:teaser], body: params[:body],
+      article_type: params[:article_type], location: params[:location], category: params[:category]
+    )
+    if current_admin.articles.last.persisted?
+      render json: {
+        message: 'The article was successfully created.'
+      }, status: 201
+    else
+      render json: {
+        message: 'Please fill out all fields.'
```

Comment:
I would've used the validations and rendered more dynamic error message depending on what is missing to save the article